.cirrus.yml

```diff
@@ -7,9 +7,8 @@ env:
 
 task:
   node_modules_cache:
-    folder: node_modules
-    fingerprint_script: cat yarn.lock
-    populate_script: yarn
+    folder: .yarn/cache
+  install_script: yarn
```

Comment:
This change has been made because Yarn 3 downloads package tarballs to `.yarn/cache` and then extracts them to `node_modules`, however n_m is platform dependent, so it's not really safe to cache it.  Instead, this just downloads the current cache, and runs `yarn` to download any new modules, unpack them, and as an added bonus, automatically remove any unused packages, so no need to re-populate the cache completely just because a dependency has changed (just let yarn modify the parts that have changed).

TL;DR: This change makes it so that yarn can properly manage the packages, and it will ensure that it cleans up after itself.